### DISCUSSION TITLE
È uscito Star Citizen... statistics

### DISCUSSION THREAD

**Discussion by Mascalian** [pdonadeo/e-uscito-joypad#60] [Ideas]
Ci pensavo dal primo giorno.
Ma mi sembrava allora una cazzata colossale, rispetto al lavoro immane di riascoltarsi tutto e creare il DB.
(a tal proposito, grazie!)

Ma ora mi viene un'idea che posso _popolare_ autonomamente:
> un grafico con la statistica della durata del pippone di Bordone per introdurre la fatidica domanda

Autonomamente: mi smazzo io tutte le puntate per misurare il tempo.

Ditemi se abbiamo modo di cacciare un grafichetto da qualche parte nel sito...

**Comment by pdonadeo** [comment]
Questa mi piace! devo solo capire come modificare lo schema del DB. Abbiamo già nel DB un dato "Osservatorio Star Citizen" col minutaggio in cui se ne parla. Questa informazione non è ancora visualizzata da nessuna parte ma c'è.

Si tratta di aggiungere la durata e poi capire cosa farci in frontend.

@Mascalian ti avviso che non è prioritario, ma creo una issue da questa discussione.
